int32 ASSISTED_BASE_NAVIGATION=1
int32 ASSISTED_ARM_NAVIGATION=2
int32 ASSISTED_GRASPING=3
